spec/factories/factories.rb

```diff
@@ -0,0 +1,47 @@
+FactoryGirl.define do
```

Comment:
You have these factories but you didn't use them in the spec file, choosing instead to declare them explicitly in the let blocks. Why?